### DISCUSSION TITLE
Mocap from video not in real-time

### DISCUSSION THREAD

**Discussion by hangeddeyvi** [endink/Mediapipe4u-plugin#152] [Q&A]
Hello,
Is it possible to animate the character in the demo project, after an mp4 video is analysed for mocap data, in order the animation to be more accurate on a PC with lower performance, and then record it a bvh?

Thanks

**Comment by endink** [comment]
https://opensource-labijie-com.translate.goog/Mediapipe4u-plugin/mocap/quick_start/anim_export.html?_x_tr_sl=zh-CN&_x_tr_tl=en&_x_tr_hl=zh-CN&_x_tr_pto=wapp

**Comment by endink** [comment]
[![Alt text](https://img.youtube.com/vi/_9j90de-XJY/0.jpg)](https://www.youtube.com/watch?v=_9j90de-XJY)

**Comment by hangeddeyvi** [comment]
And how do I ensure the correct animation without skipping the mocap data with a lower performance PC? By adjusting the frame-rate accordingly?

Thank you

**Comment by endink** [comment]
Sorry for late...

Try use fixed fps (in UE Project settings)

**Comment by endink** [comment]
You can run "stat m4u" command in UE editor to find out fps details.
